packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -178,7 +225,11 @@ class InteractiveGraphSettings extends React.Component<Props, State> {
 
     renderLabelChoices = (choices) => {
         return _.map(choices, function (nameAndValue) {
```

Comment:
Do you want to swap the `_.map(choices, ` for `choices.map(` ?  We often do that as it's a 1:1 change and reduces our use of `_`.